api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -440,159 +449,153 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	 */
 	public void deleteConceptReferenceTerm(ConceptReferenceTerm conceptReferenceTerm) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getCountOfConceptReferenceTerms(String, ConceptSource, boolean)
+	 * @see org.openmrs.api.ConceptService#getCountOfConceptReferenceTerms(String, ConceptSource, boolean)
 	 */
 	public Long getCountOfConceptReferenceTerms(String query, ConceptSource conceptSource, boolean includeRetired)
 	        throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptReferenceTerms(String, ConceptSource, Integer, Integer,
+	 * @see org.openmrs.api.ConceptService#getConceptReferenceTerms(String, ConceptSource, Integer, Integer,
 	 *      boolean)
 	 */
 	public List<ConceptReferenceTerm> getConceptReferenceTerms(String query, ConceptSource conceptSource, Integer start,
 	        Integer length, boolean includeRetired) throws APIException;
 	
 	/**
-	 * @see ConceptService#getReferenceTermMappingsTo(ConceptReferenceTerm)
+	 * @see org.openmrs.api.ConceptService#getReferenceTermMappingsTo(org.openmrs.ConceptReferenceTerm)
 	 */
 	public List<ConceptReferenceTermMap> getReferenceTermMappingsTo(ConceptReferenceTerm term) throws DAOException;
 	
 	/**
-	 * Checks if there are any {@link ConceptReferenceTermMap}s or {@link ConceptMap}s using the
-	 * specified term
-	 * 
-	 * @param term
-	 * @return true if term is in use
-	 * @throws DAOException
 	 * @should return true if a term has a conceptMap or more using it
 	 * @should return true if a term has a conceptReferenceTermMap or more using it
 	 * @should return false if a term has no maps using it
+	 * @see ConceptService#isConceptReferenceTermInUse(org.openmrs.ConceptReferenceTerm)
 	 */
 	public boolean isConceptReferenceTermInUse(ConceptReferenceTerm term) throws DAOException;
 	
 	/**
-	 * Checks if there are any {@link ConceptReferenceTermMap}s or {@link ConceptMap}s using the
-	 * specified mapType
-	 * 
-	 * @param mapType
-	 * @return true if map type is in use
-	 * @throws DAOException
 	 * @should return true if a mapType has a conceptMap or more using it
 	 * @should return true if a mapType has a conceptReferenceTermMap or more using it
 	 * @should return false if a mapType has no maps using it
+	 * @see ConceptService#isConceptMapTypeInUse(org.openmrs.ConceptMapType)
 	 */
 	public boolean isConceptMapTypeInUse(ConceptMapType mapType) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptsByName(String, Locale, Boolean)
+	 * @see org.openmrs.api.ConceptService#getConceptsByName(String, Locale, Boolean)
 	 */
 	public List<Concept> getConceptsByName(String name, Locale locale, Boolean exactLocal);
 	
 	/**
-	 * @see ConceptService#getConceptByName(String)
+	 * @see org.openmrs.api.ConceptService#getConceptByName(java.lang.String)
 	 */
 	public Concept getConceptByName(String name);
 	
 	/**
-	 * It is in the DAO, because it must be done in the MANUAL flush mode to prevent premature
```

Comment:
Are you sure this same comment is in ConceptService?
I see as losing some meaningful comments which are specific to the DAO. Not so?

Reply:
For all cases where you have replaced comments with the see annotation, ensure that we do not lose any information. In other words, ensure that were you are pointing us with the see annotation has all the information that you are removing.

Reply:
I see. :D

I have rechecked and it is not covered in ConceptService.java. I will revise in a commit. :)